### ISSUE TITLE
Add or remove users to/from lists

### ISSUE THREAD

**Issue by GrahamDo** [NicolasConstant/sengi#613]
When viewing someone's profile (whom you follow) on the default web app, you can click the hamburger menu to add or remove that person from any of your lists.

It'd be nice to be able to do that directly from within Sengi.

**Comment by NicolasConstant** [comment]
Yes, better list workflows is really something I'd like to look at, especially with the next version of mastodon that seems to add some interesting features on this side. 🙂

**Comment by spideymang** [comment]
I didnt notice this request before but I created also the same request https://github.com/NicolasConstant/sengi/issues/645 
Sorry and its a +1 for this idea
